client/me/account/main.jsx

```diff
@@ -114,7 +115,13 @@ const Account = React.createClass( {
 	},
 
 	updateColorScheme( colorScheme ) {
-		this.updateUserSetting( 'calypso_preferences.colorScheme', colorScheme );
+		const settingName = 'calypso_preferences.colorScheme';
+
+		// set a fallback color scheme if no default value is provided by the API
+		// TODO: ideally the default value should be provided on the API side
+		update( this.props.userSettings.settings, settingName, value => value || 'default' );
```

Comment:
This seems to do the trick! I'm seeing the button become active when making selections, then the option saved is persisted :)